### ISSUE TITLE
no-paragraph-content-indent rule doesn't work as expected

### ISSUE THREAD

**Issue by evilebottnawi** [wooorm/remark-lint#153]
Input:
```markdown
-   `npm run build:dev` - compile the files for development.
-   `npm run build` (alias `npm run build:prod`) - compile and optimize the files for production.
-   `npm run watch` - don’t want to manually recompile the files after every change.
```

Output:
```shell
  1:24  warning  Expected no indentation in paragraph content  no-paragraph-content-indent  remark-lint
  2:20  warning  Expected no indentation in paragraph content  no-paragraph-content-indent  remark-lint
  3:20  warning  Expected no indentation in paragraph content  no-paragraph-content-indent  remark-lint
```

Seems something wrong.

**Comment by wooorm** [comment]
Sorry about that, should be fixed now!

**Comment by evilebottnawi** [comment]
@wooorm thanks, all works good!
One thing, i have line:
```markdown
[![Build status](https://gitlab.itgalaxy.company/itgalaxy/insteria/badges/master/build.svg)](https://gitlab.itgalaxy.company/itgalaxy/insteria/commits/master) [![coverage report](https://gitlab.itgalaxy.company/itgalaxy/insteria/badges/master/coverage.svg)](https://gitlab.itgalaxy.company/itgalaxy/insteria/commits/master) [![Dependency Status](https://gemnasium.com/8a3f2fe3812670b73bb276953fe6cac5.svg)](https://gemnasium.com/17d18be2d33b7a89746221fced9ac945)
```
And get error:
```shell
1:159  warning  Expected no indentation in paragraph content  no-paragraph-content-indent  remark-lint
```

It is expected error or not? If yes, why? :smile:

**Comment by wooorm** [comment]
Found it. It’s an interesting bug, due to the image not having content (instead, having an `alt`), so the space is the first character character in the tree!
